src/praisonai/praisonai/cli/commands/managed.py

```
@@ -653,6 +653,191 @@ def ids_restore(
+                provider = resolve_compute(name)
+            except Exception:
+                # Not installed / unknown provider. Silent unless explicitly
+                # requested, where the user deserves to know it went nowhere.
+                if provider_filter:
+                    errors.append((name, "provider not installed or unknown"))
+                continue
+            try:
+                if not getattr(provider, "is_available", True):
+                    continue  # daemon down / no credentials — a normal state
+            except Exception:
+                continue
```

Ruff (0.16.1)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)